README.md

```diff
@@ -34,11 +34,13 @@ android {
 ```groovy
 dependencies {
     ...
-    implementation 'com.azure.android:azure-communication-ui-calling:1.1.0'
+    implementation 'com.azure.android:azure-communication-ui-calling:<latest stable release version>'
```

Comment:
We need to revert this change. It should state 1.1.0 and do not point to releases GitHub page

Reply:
done